scripts/uniqify-module-names.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+import shutil
+import os
+import sh
+import datetime
+
+
+parser = argparse.ArgumentParser(description="")
+parser.add_argument("--top-filelist", type=str, required=True, help="Abs path to <top>.<model>.top.f")
+parser.add_argument("--mod-filelist", type=str, required=True, help="Abs path to <top>.<model>.model.f")
+parser.add_argument("--gen-collateral-path", dest="gcpath", type=str, required=True, help="Abs path to the gen-collateral directory")
+parser.add_argument("--model-hier-json", type=str, required=True, help="Path to hierarchy JSON emitted by firtool. Must include DUT as a module.")
+parser.add_argument("--out-model-hier-json", type=str, required=True, help="Path to updated hierarchy JSON emitted by this script.")
+parser.add_argument('--dut', type=str, required=True, help='Name of the DUT module.')
+args = parser.parse_args()
+
+
+MODEL_SFX=str(datetime.date.today().year)
```

Comment:
Even though I agreed with Abe's comment about potential conflict with `_Model` as suffix, I find using the year weird. Can't it just be something like `$(MODEL).$(CONFIG)`?